website/docs/reference/exposure-properties.md

```diff
@@ -44,6 +40,7 @@ exposures:
       - ref('model')
       - ref('seed')
       - source('name', 'table')
+      - metric('metric_name')
       
     # added in dbt Core v1.3
```

Comment:
If we're VersionBlocking this entire code block to be v1.4+, we don't need the "v1.3" callout here. (I believe we did this to avoid duplicating code a few months ago, since the v1.3 changes were relatively small)